docs/test_indexing.md

```diff
@@ -0,0 +1,10 @@
+1. Create a publishing file in Alma (Either in the sandbox or production)
+    1. Create a Query that contains the data you would like to query.  Save the query after you have it correct
+    2. Change the DRDS test publishing job (Under resoruces) to point at your saved query
+    3. Run the job by clicking run under the ... menu after the save
```

Comment:
2. Resources -> Publishing Profiles, Edit the DRDS test records publishing job to point to the new data set.
3. 'Save' the dataset and 'Run'.